packages/ui-toolkit/src/utils/types/global/icon-store.d.ts

```diff
@@ -0,0 +1 @@
+declare module '@groww-tech/icon-store/mi';
```

Comment:
Shouldn't this work out of the box, why do we have to create a declaration file ?

Reply:
yeh ts error de raha tha, so add karna pada